app/build.gradle.kts

```diff
@@ -178,7 +178,7 @@ dependencies {
 
     // For KSP -> Official Annotation Processors are Not Yet Supported for KSP
     ksp("dev.zacsweers.autoservice:auto-service-ksp:1.1.0")
-    implementation("com.google.guava:guava:32.1.2-android")
+    implementation("com.google.guava:guava:32.1.3-jre")
```

Comment:
why switch from android to jre?

Reply:
was automatic, i Wii check in few minutes